system-int-test/test/cmr/system_int_test/ingest/granule_ingest_test.clj

```diff
@@ -463,6 +463,17 @@
                                (ingest/concept :granule "PROV1" "foo" :iso-smap valid-gran-metadata))]
          (is (= 201 status))))))
 
+(deftest CMR-5226-invalid-iso-smap-geographic-description-test
+  (let [coll-metadata (-> "iso-samples/CMR-5129-coll.xml" io/resource slurp)
+        invalid-gran-metadata (-> "iso-samples/invalid-CMR-5226-gran.xml" io/resource slurp)
+        _ (ingest/ingest-concept
```

Comment:
If you don't want to look at the return value move this outside of the let block.

Reply:
Done